grapher/core/Grapher.tsx

```diff
@@ -297,7 +297,9 @@ export class Grapher
     @observable.ref annotation?: Annotation = undefined
 
     @observable showFacets?: boolean = false
-    @observable lastFacet: FacetStrategy = FacetStrategy.none
+
+    // the desired faceting strategy, which might not be possible if we change the data
+    @observable selectedFacetStrategy: FacetStrategy = FacetStrategy.none
```

Comment:
Maybe we want to add this to the list of keys that get serialized into the JSON for the database? https://github.com/owid/owid-grapher/blob/c8826790803ad20a99c93cee3b628754fb7ccbff/grapher/core/GrapherInterface.ts#L117-L169

Reply:
Done